### ISSUE TITLE
Improve cluster setup instructions

### ISSUE THREAD

**Issue by cangencer** [hazelcast-guides/hazelcast-embedded-microprofile#19]
```
`You need to have a running Hazelcast cluster before using client. com.hazelcast.guide.HazelcastServer will start an instance with default configuration. When you start it multiple times, you will see Hazelcast members starting, finding each other and forming a cluster. You can also try Hazelcast CLI for starting members.```

"com.hazelcast.guide.HazelcastServer" shouldn't be shown as an alternative to start the server. You should either use 

1.) Hazelcast Cloud (link)
2.) Hazelcast Docker Image
3.) Hazelcast Download Package or CLI

**Comment by enozcan** [comment]
Let me create a template for this so other guides also benefit and we have a “Starting cluster section” standard.

**Comment by cangencer** [comment]
yes that sounds good. It's also preferable if we can inline it - so user
doesn't have to go to another page to install it.

So you can have tabs for docker, mac/os, linux etc with the specific
commands

**Comment by enozcan** [comment]
Exactly. They will be inlined just like the current templates.

**Comment by enozcan** [comment]
Closed on 9affdff by using the template https://github.com/hazelcast-guides/adoc-templates/pull/1.
